backbone-publication.sublime-project

```diff
@@ -0,0 +1,19 @@
+{
+  "folders": [{
+    "path": ".",
+    "folder_exclude_patterns": [
+      "node_modules",
+      "dist"
+    ],
+    "file_exclude_patterns": [
+    ]
```

Comment:
we should probably have a standard template for this; I like adding `package-lock.json` to this array here

Reply:
Oh yeah I'd be fine with that. Agreed a template would be nice. I always copy the project file from another Mixmax OSS project but I guess there are slight deviations